recipes/capnproto/meta.yaml

```diff
@@ -0,0 +1,45 @@
+{% set version = "0.5.3" %}
+
+package:
+  name: capnproto
+  version: {{ version }}
+
+source:
+  fn: v{{ version }}.tar.gz
+  url: https://github.com/sandstorm-io/capnproto/archive/v{{ version }}.tar.gz
+  md5: 909bd13ad6b8bc840ac78ab8f5bcb0a4
+
+build:
+  number: 0
+  skip: True  # [win and py<35]
+  features:
+    - vc14  # [win and py35]
+
+requirements:
+  build:
+    - cmake
```

Comment:
We need to add `python` as a build dependencies on Windows only for the features to work.